src/win/dtwin.c

```diff
@@ -336,43 +336,6 @@ void dtwin_set_thread_name(DWORD dwThreadID, const char *threadName)
   RaiseException(MS_VC_EXCEPTION, 0, sizeof(info) / sizeof(DWORD), (const ULONG_PTR *)&info);
 }
 
-// This is taken from: https://git.gnome.org/browse/glib/tree/gio/glocalfile.c#n2269
```

Comment:
This has been done to avoid confirmation and do a silent move to the trash. So I'm not sure it is ok to change this.

Reply:
Well, it seems GLib has been updated since then and now files go to trash completely silent.
I have tried on two Windows 10 machines.
Maybe someone else can try. Not the most urgent thing, of course